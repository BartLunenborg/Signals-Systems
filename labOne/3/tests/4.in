100 10 2.35619 10 -2.35618
