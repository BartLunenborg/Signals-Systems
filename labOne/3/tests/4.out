x(t)=14.14*cos(2*pi*100.00*t-3.14)
